axiom/nr_txn.c

```diff
@@ -60,10 +60,8 @@ struct _nr_txn_attribute_t {
 #define NR_TXN_ATTRIBUTE_TRACE_ERROR \
   (NR_ATTRIBUTE_DESTINATION_TXN_TRACE | NR_ATTRIBUTE_DESTINATION_ERROR)
 
-#define NR_TXN_ATTR(X, NAME, DESTS)                     \
-  const nr_txn_attribute_t* X = &(nr_txn_attribute_t) { \
-    (NAME), (DESTS)                                     \
-  }
+#define NR_TXN_ATTR(X, NAME, DESTS) \
+  const nr_txn_attribute_t* X = &(nr_txn_attribute_t) { (NAME), (DESTS) }
```

Comment:
Why this change? `clang-format` wants it the way it is was before this change.

Reply:
Please see: https://github.com/newrelic/newrelic-php-agent/pull/991/commits/862604b081029206c2072275588076691d30560a